intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function() {
+    if (raw_data.keys.length > 0) {
+        $("#members-chart").show();
+        var canvas = $("#members-chart")[0];
+        var data = {
+            labels: raw_data.keys,
+            datasets: [
+                {
+                    label: "Signups",
+                    backgroundColor: "rgba(151,187,205,0.5)",
+                    hoverBackgroundColor: "rgba(151,187,205,0.75)",
+                    data: raw_data.values
+                }
+            ]
+        };
+        var stepSize = 1;
+        if (capacity >= 20) {
+            stepSize = 5;
+        }
+        else if (capacity >= 100) {
+            stepSize = 10;
+        }
+        chart = new Chart(canvas.getContext("2d"), {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["chart" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2620588210/issues/source?bid=3212528&fileBranchId=3299315#l23)